src/osmf/lib/HDSPlayer.as

```diff
@@ -894,28 +898,55 @@ package
          _mediaPlayerSprite.mediaPlayer.autoDynamicStreamSwitch = false;
          //Switches to the stream with the index of bitrate (bitrate of selected bitrate ID)
          _mediaPlayerSprite.mediaPlayer.switchDynamicStreamIndex(_bitrateArray[bitrateId][1]);
-         
-         eventObject.id = _bitrateArray[bitrateId][0].id;
-         eventObject.height = _bitrateArray[bitrateId][0].height;
-         eventObject.width = _bitrateArray[bitrateId][0].width;
-         eventObject.bitrate = _mediaPlayerSprite.mediaPlayer.getBitrateForDynamicStreamIndex(_bitrateArray[bitrateId][1]) * 1000;
+         _currentBitrateId = bitrateId;
        }
        else 
        {
           _mediaPlayerSprite.mediaPlayer.autoDynamicStreamSwitch = true;
-
-          eventObject.id = "auto";
-          eventObject.height = 0;
-          eventObject.width = 0;
-          eventObject.bitrate = 0;
+          _currentBitrateId = "auto";
        }
-       dispatchEvent(new DynamicEvent(DynamicEvent.BITRATE_CHANGED,(eventObject)));
       }
       catch(error:Error)
       {
         SendToDebugger("onSetTargetBitrate Error :"+error.errorID,"onSetTargetBitrate");
       }
     }
+
+    /**
+     * Dispatched BITRATE_CHANGED event
+     * @public
+     * @method HDSPlayer#onBitrateChanged
+     * @param {DynamicStreamEvent} event This event dispatched when the properties of a DynamicStreamTrait change.
```

Comment:
Small gramatical changes

"Dispatched BITRATE_CHANGED event" 
should be:
"Dispatches BITRATE_CHANGED event"

"This event dispatched when the properties of a DynamicStreamTrait change."
should be:
"The event dispatched when the properties of a DynamicStreamTrait change."